src/test/java/com/wikia/webdriver/common/core/imageutilities/ImageComparison.java

```diff
@@ -67,4 +69,62 @@ public boolean isColorImage(BufferedImage image, Color color, int accuracy) {
     }
     return true;
   }
+
+  /**
+   * @param threshold in percentage between 0 and 100.
+   */
+  public boolean areImagesDifferent(BufferedImage image1, BufferedImage image2, int threshold) {
+    int sameCount = 0;
+    if (image1.getHeight() != image2.getHeight() || image1.getWidth() != image2.getWidth()) {
+      throw new RuntimeException("Images have different sizes");
+    }
+    int count = image1.getHeight() * image1.getWidth();
+    for (int x = 0; x < image1.getWidth(); x++) {
+      for (int y = 0; y < image1.getHeight(); y++) {
+        if (image1.getRGB(x, y) == image2.getRGB(x, y)) {
+          sameCount += 1;
+        }
+      }
+      if (sameCount > ((100 - threshold) * count) / 100D) {
+        return false;
+      }
+    }
+    return true;
+  }
+
+  public Triple getRgbVariance(BufferedImage image) {
```

Comment:
Add javadoc so in the future, everyone knows thar "middle" is green variance. Even better - make a class of image that will hold all the data for image - variance, count and make calculate the difference.